src/seedu/addressbook/data/person/UniquePersonList.java

```diff
@@ -131,4 +131,11 @@ public int hashCode() {
         return internalList.hashCode();
     }
 
+    /**
+     * Sorts all persons in the list according to their full name
+     */
+    public void sort(){
+        internalList.sort((person1, person2) -> (person1.getName().fullName.toLowerCase().compareTo(
```

Comment:
whats the difference between using `getName()` and `fullName`?

Reply:
getName() gets a Name object, and fullName, a string, is an attribute of class Name.

Reply:
and how is a `Name` object compared?